src/fiskaltrust.ifPOS/v1/me/IMESSCD.cs

```diff
@@ -0,0 +1,48 @@
+using System.ServiceModel;
+using System.Threading.Tasks;
+
+namespace fiskaltrust.ifPOS.v1.me
+{
+    /// <summary>
+    /// The interface to communicate with fiskaltrust's SCUs for Montenegro.
+    /// </summary>
+    [ServiceContract]
+    public interface IMESSCD
+    {
+        /// <summary>
+        /// Registers an electronic cash device (TCR) at the central invoice register (CIS), and returns the TCR code that is used for further operations.
+        /// </summary>
+        [OperationContract(Name = "v2/RegisterTCR")]
+        Task<RegisterTcrResponse> RegisterTcrAsync(RegisterTcrRequest registerTCRRequest);
+
+        /// <summary>
+        /// Unregisters an electronic cash device (TCR) from the central invoice register (CIS).
+        /// </summary>
+        [OperationContract(Name = "v2/UnregisterTCR")]
+        Task UnregisterTcrAsync(RegisterTcrRequest registerTCRRequest);
+
+        /// <summary>
+        /// Registers a cash deposit to the cash registers at the central invoice register (CIS).
+        /// </summary>
+        [OperationContract(Name = "v2/RegisterCashDeposit")]
+        Task<RegisterCashDepositResponse> RegisterCashDepositAsync(RegisterCashDepositRequest registerCashDepositRequest);
+
+        /// <summary>
+        /// Registers a cash withdrawal from the cash registers at the central invoice register (CIS).
+        /// </summary>
+        [OperationContract(Name = "v2/RegisterCashWithdrawal")]
+        Task<RegisterCashWithdrawalResponse> RegisterCashWithdrawalAsync(RegisterCashWithdrawalRequest registerCashDepositRequest);
```

Comment:
```suggestion
        Task RegisterCashWithdrawalAsync(RegisterCashWithdrawalRequest registerCashDepositRequest);
```
The `RegisterCashWithdrawalResponse` has no properties so we can remove it?